Woof/Woof/NetworkLayer/Endpoint/ProductionEndpoint+HTTPAPIEndpoint.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+import NetworkService
+
+///  Defines the production endpoints for the Woof app's API.
+extension ProductionEndpoint: HTTPAPIEndpoint {
+    // MARK: - Internal interface
+
+    typealias Environment = APIEnvironment
+
+    static var networkEnvironment = Environment.production
+
+    var baseURL: URL {
+        Self.baseProdURL
+    }
+
+    var path: String {
+        switch self {
+        default: return ""
+        }
+    }
+
+    var task: HTTPTask {
+        switch self {
+        default: return .request
+        }
+    }
+
+    var method: HTTPMethod {
+        switch self {
+        default: return .get
+        }
+    }
+
+    var headers: HTTPHeaders {
+        [Self.authHeader: Self.revealedKey]
+    }
```

Comment:
Should we set the headers relative toEnvironment?
```suggestion
switch Self.networkEnvironment {
        case .production:
            return [Self.authHeader: Self.revealedKey]
        case .staging:
            return [:]
        }
```

Reply:
Fixed.